### PR TITLE
chore: update cds version

### PR DESCRIPTION
**Pull request by johannes-vogel**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
         "postgres",
         "hana"
       ],
+      "dependencies": {
+        "@sap/cds": "^7.3.1"
+      },
```
Comment (johannes-vogel) [inline package-lock.json:19]:
```suggestion
```

**Comment by johannes-vogel** [comment]
@BobdenOs @patricebender please review :)

File: hana/lib/cql-functions.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const StandardFunctions = {
       ? `substring( ${x}, case when ${y} < 0 then length(${x}) + ${y} + 1 else ${y} + 1 end, ${z} )`
       : `substring( ${x}, case when ${y} < 0 then length(${x}) + ${y} + 1 else ${y} + 1 end )`,
   count: x => `count(${x || '*'})`,
+  countdistinct: x => `count(distinct ${x || '*'})`,
```
Comment (patricebender) [inline hana/lib/cql-functions.js:12]: is this intentionally part of this change? → PR title doesnt say anything bout a new function

Comment (johannes-vogel) [reply hana/lib/cql-functions.js:12]: Yes it is.

There's a change in the runtime implementation for `countdistinct` which requires this in the new HANA Service.
Bob initially created #311 but I took over this change to have only one PR.
Sorry, for not mentioning this in the initial descrpition.

Comment (patricebender) [reply hana/lib/cql-functions.js:12]: Alright, since this is only in the hana channel, it won't be necessary to add it to the changelog, yet

**Review by patricebender — APPROVED** [review]

**Pull request merged**